features/currency/managing_currency/editing_currency.feature

```diff
@@ -0,0 +1,41 @@
+@managing_currencies
+Feature: Editing currency
+    In order to change currency configuration
+    As an Administrator
+    I want to be able to edit a currency
+
+    Background:
+        Given I am logged in as an administrator
+
+    @ui
+    Scenario: Disabling a currency
```

Comment:
We used to put enable/disable scenarios to different feature file and name it `toggling_`resource name